Add links to Pix payment samples and adjust card payment samples description (#1394)
mercadopago/devsite-docs: guides/online-payments/checkout-api/other-payment-ways.en.md

You can receive payment immediately with Pix from any bank or digital wallet using QR code or payment code.

> GIT
>
> Pix integration examples
>

> > We have [complete integration examples](https://github.com/mercadopago/pix-payment-sample) available on GitHub so you can download it immediately.

Reviewer: ```suggestion
> You can find [complete integration examples](https://github.com/mercadopago/pix-payment-sample) on GitHub so you can download them right away.
```
